src/composable/orderTypes/Twap.spec.ts

```diff
@@ -97,6 +101,37 @@ describe('Twap.fromData', () => {
   })
 })
 
+describe('Id', () => {
+  test('Id is computed correctly', () => {
+    const twap = Twap.fromData({ ...TWAP_PARAMS_TEST }, SALT)
+    expect(twap.id).toEqual(TWAP_ID)
+  })
+
+  test("Id doesn't change for the same params and salt", () => {
+    const twap1 = Twap.fromData({ ...TWAP_PARAMS_TEST }, SALT)
+    const twap2 = Twap.fromData({ ...TWAP_PARAMS_TEST }, SALT)
+
+    expect(twap1.id).toEqual(twap2.id)
+  })
+
+  test('Id changes for different params and same salt', () => {
+    const twap1 = Twap.fromData({ ...TWAP_PARAMS_TEST }, SALT)
+    const twap2 = Twap.fromData({ ...TWAP_PARAMS_TEST }, SALT_2)
+
+    expect(twap1.id).not.toEqual(twap2.id)
+    expect(twap2.id).toEqual(TWAP_ID_2)
+  })
+
+  test('Id changes for same params and different salt', () => {
+    const twap = Twap.fromData(
+      { ...TWAP_PARAMS_TEST, startTime: { startType: StartTimeValue.AT_EPOCH, epoch: BigNumber.from(123456789) } },
+      SALT
+    )
```

Comment:
Here the params are different and the salt is the same, contrary to the test description

Reply:
the test desc were swapped